src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -28,49 +29,67 @@ export function LoadingScreenTemplate({ children }: { children?: ReactNode }) {
   }, [hoverboardColor]);
 
   return (
-    <BubbleViewportWrapper>
-      <div className="relative flex w-96 flex-col items-center space-y-1 rounded-lg bg-loadingBoxes p-4 py-12 shadow-sm">
-        <div className="flex flex-col items-center space-y-2">
-          <div className="h-32 w-32" onClick={changeHoverboardColor}>
-            <Hoverboard color={hoverboardColor} />
+    <div className={styles.loadingScreenWrapper}>
+      <BubbleViewportWrapper>
+        <div className={styles.loadingScreenWrapper}>
+          <div className="flex flex-col items-center space-y-2">
+            <div className={styles.hoverboardWrapper} onClick={changeHoverboardColor}>
+              <Hoverboard color={hoverboardColor} />
+            </div>
+            {children}
           </div>
-          {children}
         </div>
-      </div>
-    </BubbleViewportWrapper>
+      </BubbleViewportWrapper>
+    </div>
   );
 }
 
+const phrases = [
+  "Tuning the flux capacitor...",
+  "Prepping DeLorean for time travel...",
+  "Revving up to 88 miles per hour...",
+  "Gathering 1.21 gigawatts of power...",
+  "Ensuring temporal paradox safeguards...",
+  "Being careful not to mess up your timeline...",
```

Comment:
Can we go further? https://sharegpt.com/c/G7i464K

At some point we should add a section to our language guide around how we discuss time travel, but it's probably too early for that. At this stage, I love seeing how experimental we can be

Reply:
Love this idea, will play with it in a separate PR away from the code changes Brian's suggesting.